binderhub/static/js/src/image.js

```diff
@@ -1,3 +1,7 @@
+import { NativeEventSource, EventSourcePolyfill } from 'event-source-polyfill';
+
+const EventSource = NativeEventSource || EventSourcePolyfill;
```

Comment:
Do we also need to set it globally? It is a hard question to answer perhaps. Should we go for with or without setting it globally you think?

I'm really not sure at all, so this suggestion is just for your convenience rather than an actual "I think this should be done".

```suggestion
const EventSource = NativeEventSource || EventSourcePolyfill;
global.EventSource = EventSource;
```

Reply:
It doesn't need to be set globally. It's used explicitly only in that file, and default lexical scope of `const` means this works as it should.